egs/librimix/ConvTasNet/utils/parse_options.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
It doesn't look like a symlink but that's fine for now, I have to do it for all the recipes, it's not consistent for now.